### PR TITLE
Update UpgradesGUI.java

### PR DESCRIPTION
**Pull request by sh0inx**

### CODE REVIEW

File: src/main/java/com/iridium/iridiumteams/gui/UpgradesGUI.java [hunk 1]
```diff
@@ -56,14 +56,21 @@ public void addContent(Inventory inventory) {
             int hours = Math.max((int) (teamEnhancement.getRemainingTime() / 3600), 0);
             String nextLevel = nextData == null ? "N/A" : String.valueOf(teamEnhancement.getLevel() + 1);
             String cost = nextData == null ? "N/A" : String.valueOf(nextData.money);
+
+            String bankCosts = "";
+            for(Double bankCost : nextData.bankCosts.values()) {
+                bankCosts = nextData == null ? "N/A" : bankCosts + " " + bankCost;
```
Comment (MertUnverdi) [inline src/main/java/com/iridium/iridiumteams/gui/UpgradesGUI.java:62]: What u did at here ? These is just numbers?

**Review by MertUnverdi — CHANGES_REQUESTED** [review]
Could use BankItem#getName ?

File: src/main/java/com/iridium/iridiumteams/gui/UpgradesGUI.java (hunk 1 above)
Comment (sh0inx) [reply src/main/java/com/iridium/iridiumteams/gui/UpgradesGUI.java:62]: yeah i havent figured out adding the name yet, im probably going to do that next

Comment (sh0inx) [reply src/main/java/com/iridium/iridiumteams/gui/UpgradesGUI.java:62]: preferably id like to add a config option to determine how that's formatted

Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumteams/gui/UpgradesGUI.java:62]: Just add a displayName for bank items (this also can be used on placeholders or smthing)

For now:
`StringBuilder bankCosts = new StringBuilder();
for (Map.Entry<String, Double> bankCost : nextData.bankCosts.entrySet()) {
    bankCosts.append(WordUtils.capitalizeFully(bankCost.getKey().toLowerCase())).append(": ").append(bankCost.getValue().toString());
}`

**Comment by PeachesMLG** [comment]
We also need this change in the BoostersGUI, id also be keen to change the placeholders slightly and have a different one for each bank item?

e.g. bank_crystals_cost bank_money_cost bank_experience_cost ect

**Comment by sh0inx** [comment]
> We also need this change in the BoostersGUI, id also be keen to change the placeholders slightly and have a different one for each bank item?
> 
> e.g. bank_crystals_cost bank_money_cost bank_experience_cost ect

i like this plan, ill see what i can do

**Comment by sh0inx** [comment]
closing this because its been implemented *correctly* in [IT #49](https://github.com/Iridium-Development/IridiumTeams/pull/49)

**Pull request closed without merging**